src/app/app-routing.module.ts

```diff
@@ -13,6 +13,14 @@ export const routes: Routes = [
     pathMatch: 'full'
   },
   {
+    path: 'home',
```

Comment:
@pmuir why did you add a Home route? The first route takes you to Home. If you want a direct path then you should use the Login example. The style you used here is for lazy loading.